src/main/resources/application.properties

```diff
@@ -1,4 +1,12 @@
-spring.datasource.url=jdbc:postgresql://localhost/didresolver
+spring.datasource.url=jdbc:postgresql://localhost/postgres
 spring.datasource.username=postgres
 spring.datasource.password=sora_1234
 spring.datasource.driver-class-name=org.postgresql.Driver
+# consider changing this if we are using different databases
+didresolver.datasource.url=jdbc:postgresql://localhost/postgres
+didresolver.datasource.username=postgres
+didresolver.datasource.password=sora_1234
+
+iroha.datasource.url=jdbc:postgresql://localhost/postgres
+iroha.datasource.username=postgres
+iroha.datasource.password=sora_1234
```

Comment:
is it really necessary to duplicate connection? 

Why are there didresolver and iroha datasources (and also `spring`)?

Reply:
I had doubts after I made this... I agree, this is an overkill to have multiple datasources for now. Removing it